config/routes.rb

```diff
@@ -1,9 +1,10 @@
 Rails.application.routes.draw do
-  namespace :api do
-    namespace :v1 do
-      resources :todos
-    end
-  end
```

Comment:
As this change wasn't necessary to the tests should i have left this out of the pull request?

Reply:
No, this is still part of the PR to make the feature functional.

You can test based on routes as well in Rails to be more comprehensive.

Reply:
Ah, wait, this was a removal.

This is more of a personal preference type of thing. It was some minor cleanup. Purist will tell you to do it in a separate PR. I'm personally fine with some added cleanup in a PR.